templates/zerver/help/add-custom-emoji.md

```diff
@@ -3,8 +3,10 @@
 If enabled by your Zulip organization administrator, you can add custom
 emojis to your organization for other members to use.
 
-1. Go to the [Emoji Settings](/#organization/emoji-settings)
-{!admin.md!}
+{setting(Custom emoji)}
```

Comment:
https://github.com/zulip/zulip/pull/5662/files has the macro !icon(icon_name), while the [other](https://github.com/zulip/zulip/blob/master/zerver/lib/bugdown/api_arguments_table_generator.py) [macros](https://github.com/zulip/zulip/blob/master/zerver/lib/bugdown/api_code_examples.py) in the folder have the format {macroname|arg1|arg2}. This conforms to the standard that is already in the codebase.

Reply:
Almost. *sigh* *facepalm* I should change it to {setting|Custom emoji}. I'll do that once I get comments on the rest of the PR.